corehq/apps/reports/export.py

```diff
@@ -89,7 +90,7 @@ def generate_bulk_file(self):
                                 separator=self.separator)
                         ])]
                     )
-
+                update_progress(i+1)
```

Comment:
Ah, got it. At first glance I though that `total = sum([len(file.export_objects) for file in bulk_export_helper.bulk_files]) + 1` was the total number of documents.